glue_jupyter/bqplot/common/tools.py

```diff
@@ -519,7 +544,11 @@ def update_selection(self, *args):
                 rx = abs(x[1] - x[0]) * 0.5
                 ry = abs(y[1] - y[0]) * 0.5
                 outer_r = float(rx + ry) * 0.5
-                if self._roi is None:
+                # Avoid drawing invisible shape.
+                if isinstance(self._roi, CircularAnnulusROI) and outer_r < 2:
+                    outer_r = self._roi.outer_radius
+                    inner_r = self._roi.inner_radius
+                elif self._roi is None:
```

Comment:
```suggestion
                # Avoid drawing invisible shape.
                outer_r = max(float(rx + ry) * 0.5, 2.0)
```
Should really just replace `outer_r` above the deleted line.